server/src/db/migrations/20210223170655-create-services.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Services', {
+      id: {
+        primaryKey: true,
+        type: Sequelize.STRING,
+        allowNull: false,
+        defaultValue: Sequelize.literal('uuid_generate_v4()'),
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      itemInFront: {
+        type: Sequelize.INTEGER,
```

Comment:
item in front will be a reference to an id of another row within the table. My fault for not clear enough ac / not having that meeting sooner